app/src/main/kotlin/uk/govuk/app/AppViewModel.kt

```diff
@@ -30,29 +31,46 @@ internal class AppViewModel @Inject constructor(
     configRepo: ConfigRepo,
     flagRepo: FlagRepo,
     private val analytics: Analytics
-): ViewModel() {
+) : ViewModel() {
 
     private val _uiState: MutableStateFlow<AppUiState?> = MutableStateFlow(null)
     val uiState = _uiState.asStateFlow()
 
     init {
         viewModelScope.launch {
-            val isConfigInitSuccess = configRepo.initConfig()
-            if (!isConfigInitSuccess) {
-                // Todo - handle config failure - specific message for network connectivity issue???
+            val configResult = configRepo.initConfig()
+            configResult.onSuccess {
+                _uiState.value = AppUiState(
+                    shouldDisplayAppUnavailable = !flagRepo.isAppAvailable(),
+                    shouldDisplayForcedUpdate = flagRepo.isForcedUpdate(BuildConfig.VERSION_NAME),
+                    shouldDisplayRecommendUpdate = flagRepo.isRecommendUpdate(BuildConfig.VERSION_NAME),
+                    shouldDisplayAnalyticsConsent = analytics.isAnalyticsConsentRequired(),
+                    shouldDisplayOnboarding = flagRepo.isOnboardingEnabled() && !appRepo.isOnboardingCompleted(),
+                    shouldDisplayTopicSelection = flagRepo.isTopicsEnabled() && !appRepo.isTopicSelectionCompleted(),
+                    isSearchEnabled = flagRepo.isSearchEnabled(),
+                    isRecentActivityEnabled = flagRepo.isRecentActivityEnabled(),
+                    isTopicsEnabled = flagRepo.isTopicsEnabled()
+                )
```

Comment:
This is getting a bit of beast 🐻

Reply:
Not a blocker, just something we might want to look at if it needs to grow more.

Reply:
Done (I hope is what you're expecting?)